src/components/ContentArea.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function ContentArea() {
```

Comment:
Created a `ContentArea` component because I believe that that there will likely be more contents later on.

Reply:
I think this too generic of a name since other pages will have content and this makes it less reusable, why don't you name it `ArticleFeed` or something similar.

Reply:
Addressed. The component name has been changed to `ArticleFeed`. Thanks!